tools/mo/openvino/tools/mo/utils/check_ie_bindings.py

```diff
@@ -70,7 +70,7 @@ def import_core_modules(silent: bool, path_to_module: str):
         print("{}: \t{}".format("Model Optimizer version", mo_version))
 
         versions_mismatch = False
-        if mo_version != ie_version:
+        if mo_version == ie_version:
```

Comment:
this is not correct for mismatch case. please fix it

Reply:
Thanks for noticing. Fixed